src/crab/ebpf_domain.hpp

```diff
@@ -376,6 +376,18 @@ class ebpf_domain_t final {
     // a constraint, so all of them would be converted to some kind of
     // constraint that is added to the domain.
 
+    void operator()(const basic_block_t& bb) {
+        for (const Instruction& statement : bb) {
+            std::visit(*this, statement);
+        }
+        add(variable_t::instruction_count(), z_number((unsigned)bb.size()));
+    }
+
+    bool terminates() {
+        using namespace crab::dsl_syntax;
+        return m_inv.entail(variable_t::instruction_count() <= 100000);
```

Comment:
Suggest using a #define instead of hard coding 100000 inside this function

Reply:
Of course, it will not stay this way.